lib/headquarters/request.rb

```diff
@@ -8,8 +8,8 @@ class Request
     def self.perform_with_auth(http_method, path, params = {}, options = {})
       options_with_auth = options.merge(
         basic_auth: {
-          username: ENV['BASIC_AUTH_USER'],
-          password: ENV['BASIC_AUTH_PASS']
+          username: ENV['HQ_BASIC_AUTH_USER'] || ENV['BASIC_AUTH_USER'],
```

Comment:
Then we should probably add a deprecation message. For the case when `BASIC_AUTH_USER` is used, and in preparation for the move to oauth.